### PR TITLE
Add host networking doc

### PR DESCRIPTION
**Pull request by zshi-redhat**
Signed-off-by: Zenghui Shi <zshi@redhat.com>

Related-Issue: https://issues.redhat.com/browse/USHIFT-640
Related-Issue: https://issues.redhat.com/browse/USHIFT-663

### CODE REVIEW

**Comment by ggiguash** [comment]
@zshi-redhat, could you add a reference to this doc from the top-level README.md?

File: docs/network/host_networking.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
+# Host Networking in MicroShift
+
+## Table of Contents
```
Comment (ggiguash) [inline docs/network/host_networking.md:3]: I thought TOC was generated automatically. Can we skip this section?

Comment (zshi-redhat) [reply docs/network/host_networking.md:3]: @ggiguash What wiki format do you use to auto-generate the TOC? I tried the following, but doesn't work:
```
[TOC]
{{TOC left}}
```

Comment (ggiguash) [reply docs/network/host_networking.md:3]: I think it's generated automatically when viewed in GitHub. 
image

File: docs/network/host_networking.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+**NOTE:** These IPs shall be reserved in user deployment.
+
+## Interfaces
+
+The following network interfaces are created or modified by ovn-kubernetes by default:
+
+|name               |type              |description        |comment                                                                                             |
+|:------------------|:-----------------|:------------------|:---------------------------------------------------------------------------------------------------|
+|br-ex              |OVS bridge        |gateway bridge     |created by microshift-ovs-init.service or manually (disableOVSInit = true)                          |
```
Comment (mangelajo) [inline docs/network/host_networking.md:50]: should we also talk about br-ex1 ? may be another PR

File: docs/network/host_networking.md (hunk 1 above)
Comment (zshi-redhat) [reply docs/network/host_networking.md]: fixed.

File: docs/network/host_networking.md (hunk 2 above)
Comment (zshi-redhat) [reply docs/network/host_networking.md:43]: good catch, will fix it.

Comment (zshi-redhat) [reply docs/network/host_networking.md]: added br-ex1.

**Comment by zshi-redhat** [comment]
> @zshi-redhat, could you add a reference to this doc from the top-level README.md?

added.
